charts/base/templates/daemonset.yaml

```diff
@@ -95,8 +96,37 @@ spec:
                   key: LOKI_BEARER_TOKEN
             {{- end}}
             {{- end}}
-        - name: nullplatform-logging-metrics-exporter
-          image: {{ .Values.logging.metricsExporter.image }}
+        - name: nullplatform-controlplane-agent
+          image: {{ .Values.logging.agent.image }}
+          command:
+            - /app/k8s_launcher.sh
+          env:
+            - name: NODE_NAME
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: spec.nodeName
+            - name: NULLPLATFORM_APIKEY
+              valueFrom:
+                secretKeyRef:
+                  key: NULLPLATFORM_APIKEY
+                  name: nullplatform-secret
+            # Not sure if this is required, I'd leave this out for now 
```

Comment:
@geisbruch validate if this is needed in the context of Azure at least

Reply:
Not required for azure and also I think we can omit it for aws, we should do double check why is in use I think It's for self discovery, in K8S may we should use an SA